packages/address/src/address.test.ts

```diff
@@ -145,7 +147,12 @@ describe('Address utils', () => {
   });
 
   test('toB256 (b256 to b256)', () => {
-    expect(() => utils.toB256(ADDRESS_B256 as Bech32Address)).toThrow();
+    const address = ADDRESS_B256 as Bech32Address;
+    const expectedError = new FuelError(
+      FuelError.CODES.INVALID_BECH32_ADDRESS,
+      `Invalid Bech32 Address: ${address}`
```

Comment:
Would be nice if the message was derived from the code 🤔

Reply:
It seems reasonable to me that we currently focus primarily on the `ErrorCode`s and leave the error messages as they are. After we go through all packages with `FuelError`, `ErrorCode` patterns will start to emerge and we'll standardize them a bit more, as one `ErrorCode` can be used for multiple errors with different messages. That's why I'd leave it as it currently is and return to it afterwards.

Reply:
@danielbate What do you mean when you say the message should be **_derived_** from the code?

Reply:
Yes, messages will probably only sometimes derive from the code.

We can review this as we go.